CHANGELOG.md

```diff
@@ -3,9 +3,9 @@
 ## Not released
 - Client Side Widgets [#159](https://github.com/CartoDB/carto-react-template/pull/159)
 - Fix basic tab layout id broken on first load [#162](https://github.com/CartoDB/carto-react-template/issues/162)
-- Add sources and views selector in hygen layer generator [#165](https://github.com/CartoDB/carto-react-template/pull/165)
-- Add hygen source generator [#165](https://github.com/CartoDB/carto-react-template/pull/165)
-- Refactor source structure for simplification [#164](https://github.com/CartoDB/carto-react-template/issues/164)
+- Add sources and views selector in hygen layer generator. [#165](https://github.com/CartoDB/carto-react-template/pull/165)
```

Comment:
Could you please remove the end points to be homogeneous with the rest of entries?